lib/src/common/variables.ts

```diff
@@ -284,8 +284,14 @@ export const componentTokens = {
     buttonPaddingBottom: "0px",
     buttonPaddingLeft: "16px",
     buttonPaddingRight: "16px",
+    buttonHeight: "40px",
+    buttonBorderRadius: "4px",
+    buttonBorderStyle: CoreTokens.border_none,
+    buttonBorderThickness: CoreTokens.border_width_0,
+    buttonBorderColor: CoreTokens.color_transparent,
     disabledColor: CoreTokens.color_grey_500,
     disabledButtonBackgroundColor: CoreTokens.color_transparent,
+    disabledButtonBorderColor: CoreTokens.color_transparent,
```

Comment:
Adding new Component Tokens may require updates in advanced-theme / advanced-schema

Reply:
Those are in website, unless these changes are merged we can not change the ones in website